apps/storybook-e2e/src/e2e/components/http-error-message/vl-http-error-message.stories.cy.ts

```diff
@@ -1,15 +1,180 @@
+    'http://localhost:8080/iframe.html?args=&id=components-http-error-message--http-error-message-400&viewMode=story';
+const httpErrorMessage401Url =
+    'http://localhost:8080/iframe.html?args=&id=components-http-error-message--http-error-message-401&viewMode=story';
+const httpErrorMessage403Url =
+    'http://localhost:8080/iframe.html?args=&id=components-http-error-message--http-error-message-403&viewMode=story';
+const httpErrorMessage404Url =
+    'http://localhost:8080/iframe.html?args=&id=components-http-error-message--http-error-message-404&viewMode=story';
+const httpErrorMessage405Url =
+    'http://localhost:8080/iframe.html?args=&id=components-http-error-message--http-error-message-405&viewMode=story';
+const httpErrorMessage408Url =
+    'http://localhost:8080/iframe.html?args=&id=components-http-error-message--http-error-message-408&viewMode=story';
+const httpErrorMessage410Url =
+    'http://localhost:8080/iframe.html?args=&id=components-http-error-message--http-error-message-410&viewMode=story';
+const httpErrorMessage411Url =
+    'http://localhost:8080/iframe.html?args=&id=components-http-error-message--http-error-message-411&viewMode=story';
+const httpErrorMessage412Url =
+    'http://localhost:8080/iframe.html?args=&id=components-http-error-message--http-error-message-412&viewMode=story';
+const httpErrorMessage413Url =
+    'http://localhost:8080/iframe.html?args=&id=components-http-error-message--http-error-message-413&viewMode=story';
+const httpErrorMessage414Url =
+    'http://localhost:8080/iframe.html?args=&id=components-http-error-message--http-error-message-414&viewMode=story';
+const httpErrorMessage415Url =
+    'http://localhost:8080/iframe.html?args=&id=components-http-error-message--http-error-message-415&viewMode=story';
+const httpErrorMessage500Url =
+    'http://localhost:8080/iframe.html?args=&id=components-http-error-message--http-error-message-500&viewMode=story';
+const httpErrorMessage501Url =
+    'http://localhost:8080/iframe.html?args=&id=components-http-error-message--http-error-message-501&viewMode=story';
+const httpErrorMessage502Url =
+    'http://localhost:8080/iframe.html?args=&id=components-http-error-message--http-error-message-502&viewMode=story';
+const httpErrorMessage503Url =
+    'http://localhost:8080/iframe.html?args=&id=components-http-error-message--http-error-message-503&viewMode=story';
+const httpErrorMessage504Url =
+    'http://localhost:8080/iframe.html?args=&id=components-http-error-message--http-error-message-504&viewMode=story';
+const httpErrorMessage505Url =
+    'http://localhost:8080/iframe.html?args=&id=components-http-error-message--http-error-message-505&viewMode=story';
+const httpErrorMessage506Url =
+    'http://localhost:8080/iframe.html?args=&id=components-http-error-message--http-error-message-506&viewMode=story';
+
+describe('story vl-http-error-message - default', () => {
+    it('the http error message is rendered', () => {
```

Comment:
We gebruiken doorgaans iets in de trant van `should render http error message`. Kan je dit eventueel find-en-replacen?

Reply:
done